cluster/images/etcd/migrate/options.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	supportedEtcdVersions = []string{"3.0.17", "3.1.12", "3.2.24", "3.3.17", "3.4.13"}
+	supportedEtcdVersions = []string{"3.0.17", "3.1.12", "3.2.24", "3.3.17", "3.4.13", "3.5.0-beta.3"}
```

Comment:
will we drop the pre-release version from here when we bump to 3.5.0?

Reply:
sounds good, please open an issue to track bumping this and link it here so we can put it in the 1.22 milestone and ensure it is resolved before release

Reply:
Done, https://github.com/kubernetes/kubernetes/issues/102294